<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgASCCDL</class>
 <widget class="QWidget" name="WdgASCCDL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item column="0" row="3">
    <widget class="KisColorButton" name="btnSlope">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item column="2" row="3">
    <widget class="KisColorButton" name="btnPower">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item column="1" row="1">
    <widget class="QLabel" name="lblOffset">
     <property name="text">
      <string>Offset:</string>
     </property>
    </widget>
   </item>
   <item colspan="2" column="0" row="0">
    <widget class="QLabel" name="lblBase">
     <property name="text">
      <string>ASC-CDL  color balance</string>
     </property>
    </widget>
   </item>
   <item column="1" row="3">
    <widget class="KisColorButton" name="btnOffset">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item column="2" row="1">
    <widget class="QLabel" name="lblPower">
     <property name="text">
      <string>Power:</string>
     </property>
    </widget>
   </item>
   <item column="0" row="1">
    <widget class="QLabel" name="lblSlope">
     <property name="text">
      <string>Slope:</string>
     </property>
    </widget>
   </item>
   <item column="0" row="2">
    <widget class="KisVisualColorSelector" name="slopeSelector" native="true"/>
   </item>
   <item column="1" row="2">
    <widget class="KisVisualColorSelector" name="offsetSelector" native="true"/>
   </item>
   <item column="2" row="2">
    <widget class="KisVisualColorSelector" name="powerSelector" native="true"/>
   </item>
   <item colspan="3" column="0" row="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisColorButton</class>
   <extends>QPushButton</extends>
   <header>kis_color_button.h</header>
  </customwidget>
  <customwidget>
   <class>KisVisualColorSelector</class>
   <extends>QWidget</extends>
   <header>KisVisualColorSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>